scaleway/data_source_vpc_private_network.go

```diff
@@ -25,6 +26,11 @@ func dataSourceScalewayVPCPrivateNetwork() *schema.Resource {
 		ValidateFunc:  validationUUIDorUUIDWithLocality(),
 		ConflictsWith: []string{"name"},
 	}
+	dsSchema["is_regional"] = &schema.Schema{
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Description: "Whether this is a regional or zonal private network",
+	}
 
 	return &schema.Resource{
```

Comment:
Outside of current code change but `zone` and `region` cannot be configured for this datasource.
Those fields should be added to `addOptionalFieldsToSchema`